packages/components/src/font-size-picker/README.md

```diff
@@ -10,57 +10,79 @@ The component renders a series of buttons that allow the user to select predefin
 import { FontSizePicker } from '@wordpress/components';
 import { withState } from '@wordpress/compose';
 
+...
 const MyFontSizePicker = withState( {
 	fontSize: 16,
-} )( ( { fontSize, setState } ) => { 
+} )( ( { fontSize, setState } ) => {
 	const fontSizes = [
-		{ shortName: 'S', size: 12 },
-		{ shortName: 'M', size: 16 }
+		{ name: 'Small', size: 12 },
+		{ name: 'Big', size: 26 },
 	];
 	const fallbackFontSize = 16;
-	
-	return ( 
-		<FontSizePicker 
-			fontSizes={ fontSizes } 
+
+	return (
+		<FontSizePicker
+			fontSizes={ fontSizes }
 			value={ fontSize }
 			fallbackFontSize={ fallbackFontSize }
-			onChange={ fontSize => setState( { fontSize } ) } 
+			onChange={ ( newFontSize ) => {
+				setState( { fontSize: newFontSize } );
+			} }
 		/>
-	); 
+	);
 } );
+
+...
+
+<MyFontSizePicker />
 ```
 
 ## Props
 
 The component accepts the following props:
 
-### fontSizes
+### disableCustomFontSizes
 
-An array of font size objects. The object should contain properties size, name, shortName.
-The property "size" contains a number with the font size value. The "shortName" property includes a small label used in the buttons. Property "name" is used as the label when shortName is not provided.
+If true it will not be possible to choose a custom fontSize, the user will be forced to pick one of the sizes passed in fontSizes.
 
-- Type: `Array`
-- Required: No
+- Type: `Boolean`
+- Required: no
+- Default: `false`
 
 ### fallbackFontSize
 
-In no value exists this prop contains the font size picker slider starting position.
+In no value exists this prop contains the font size picker slider starting position. Only relevant if withSlider is true.
 
 - Type: `Number`
 - Required: No
 
-### value
+### fontSizes
 
-The current font size value. If a button value matches the font size value that button is pressed. RangeControl is rendered with this value.
+An array of font size objects. The object should contain properties size and name.
```

Comment:
Do you also need to provide a "slug"? I might be wrong but I think it's expected in terms of generating CSS classes (although maybe that doesn't get passed into this version of the component?)

I'd also like to see an example here… happy to add that, if you want. Something simple like

```
{
    size: 12,
    name: "My Font Size",
    slug: "font-slug" // if needed
}
```

Reply:
(I know we show it above, but being more explicit will be good for users who are newer to JS.)

Reply:
You are totally right I missed the slug property and it is used to generate classes in the component. The situation was resolved. thank you for catching this.

Reply:
@jorgefilipecosta No prob! I also thought that it was possible we only used `slug` inside the editor version of this component so I'm glad I checked :)